### DISCUSSION TITLE
Implementation of blocked system call

### DISCUSSION THREAD

**Discussion by ytrewq271828** [ANLAB-KAIST/KENSv3#175] [Project Q&A 2. Transport Layer (TCP)]
How can we block the user application?
I think it is necessary in implementing accept() properly.
I searched for the documentations but there was no clear answer. 

Thanks in advance.

**Comment by Pusnow** [comment]
You can block an application by *not* calling `returnSystemCall` method.

https://github.com/ANLAB-KAIST/KENSv3/wiki/Tip:-Returning-a-System-Call
